draft-ietf-quic-recovery.md

```diff
@@ -834,19 +834,19 @@ DetectLostPackets(largest_acked):
   foreach (unacked < largest_acked.packet_number):
     time_since_sent = now() - unacked.time_sent
     delta = largest_acked.packet_number - unacked.packet_number
-    if (time_since_sent > delay_until_lost):
-      lost_packets.insert(unacked)
-    else if (delta > reordering_threshold)
+    if (time_since_sent > delay_until_lost ||
+      delta > reordering_threshold):
```

Comment:
I think the convention is to indent the second line of the condition 2 more than the first?

Reply:
Isn't that what I did?